libs/ledger-live-common/src/cal/partners.test.ts

```diff
@@ -58,10 +61,12 @@ describe("transformData", () => {
     ],
   ])(
     "should transform providers data correctly with %p env",
-    (env: string, expected: Record<string, ExchangeProvider>) => {
+    (_env: string, testConfig: boolean, expected: Record<string, ExchangeProvider>) => {
+      setEnv("MOCK_EXCHANGE_TEST_CONFIG", testConfig);
```

Comment:
Don't add env variable setting in this test.
The method you test shouldn't rely on env variable.

the `testConfig` is useless as it is redundant with the `env` parameter.

Reply:
env do not have sense anymore, only set the title of the test.